package.json

```diff
@@ -109,7 +109,7 @@
     "webpack-cli": "^3.3.2"
   },
   "peerDependencies": {
-    "amazon-chime-sdk-js": "1.x || 2.x",
```

Comment:
Hm.. We don't want to force customers to use only 2.4. We really want 2.4 and up, but now that I think about it, I'm unsure if you can specify that. We may have to just do 2.x here

Reply:
Good links! Updated.

Reply:
`>=2.4.x` would upgrade to the next major version (3.x, 4.x, etc), which could break builders' applications as well

I think we want something like this? (Probably good to double check the link I sent you)

`>=2.4 <3.0`

Maybe @davejlane can confirm if this is the right approach

Reply:
Ok, I'm surprised we would need to worry about 3.0 before it exists, but I can take a deeper look into the standards in those docs.